src/main/java/frc/robot/subsystems/IntakeSystem.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class IntakeSystem extends SubsystemBase {
+    private CANSparkMax motor;
+    private DigitalInput limitSwitch;
```

Comment:
```suggestion
    private final DigitalInput limitSwitch;
```
Use `final` modifier for class members when possible to avoid assignment bugs.

Reply:
done